dotcom-rendering/src/components/HeaderTopBarMyAccount.stories.tsx

```diff
@@ -3,7 +3,7 @@ import { MyAccount } from './HeaderTopBarMyAccount';
 
 export default {
 	component: MyAccount,
-	title: 'Components/MyAccount',
+	title: 'Components/HeaderTopBarMyAccount',
```

Comment:
Wonder if it's worth having a Components/Masthead subfolder or some such with one eye on the other sibling components in the works? Appreciate this may go against convention but figure it's worth floating here